Libraries/Components/TabBarIOS/TabBarIOS.ios.js

```diff
@@ -27,6 +27,9 @@ class TabBarIOS extends React.Component {
     barTintColor?: $FlowFixMe,
     translucent?: boolean,
     itemPositioning?: 'fill' | 'center' | 'auto',
+    onTVFocus?: $FlowFixMe,
```

Comment:
Let's keep these here for now, we'll see how the other tv-specific properties scale.